src/wallet/wallet.cpp

```diff
@@ -8,6 +8,7 @@
 #include <chain.h>
 #include <consensus/consensus.h>
 #include <consensus/validation.h>
+#include <external_signer.h>
```

Comment:
edaff3958e3c9d92e7aa15265ecad155b05714bf: I might make a separate commit for this fix